src/components/Tutorial/Tutorial2StepContext1SubThought.js

```diff
@@ -28,9 +28,8 @@ const Tutorial2StepContext1SubThought = ({ cursor, tutorialChoice, rootSubthough
     }</p>
     {
       // e.g. Home
-      rootSubthoughts.find(child => child.value.toLowerCase() === TUTORIAL_CONTEXT1_PARENT[tutorialChoice].toLowerCase()) &&
-        // e.g. Home/To Do
-        getChildrenRanked(store.getState(), [TUTORIAL_CONTEXT1_PARENT[tutorialChoice]]).find(child => child.value.toLowerCase() === TUTORIAL_CONTEXT[tutorialChoice].toLowerCase())
+      // e.g. Home/To Do
+      getChildrenRanked(store.getState(), [TUTORIAL_CONTEXT1_PARENT[tutorialChoice]]).find(child => child.value.toLowerCase() === TUTORIAL_CONTEXT[tutorialChoice].toLowerCase())
```

Comment:
If you remove the `rootSubthoughts` check, then it's possible that the parent no longer exists, so it wouldn't be appropriate to ask them to go to the next step.